src/dom/html.js

```diff
@@ -17,7 +17,7 @@ import { each } from '../util';
 
 export const html = function(fragment) {
 
-  if(typeof fragment !== 'string') {
+  if(['string', 'number'].indexOf(typeof fragment) < 0) {
```

Comment:
Please test for `fragment === undefined` (simpler and more consistent with rest of codebase)

Reply:
on my way

Reply:
done